internal/host/validator.go

```diff
@@ -674,50 +676,50 @@ func (v *validator) hasSufficientNetworkLatencyRequirementForRole(c *validationC
 		return ValidationSuccess
 	}
 
+	v.log.Debugf("Maximum network latency allowed for host %s: %f", c.host.ID, *c.clusterHostRequirements.Total.NetworkLatencyThresholdMs)
 	if len(c.host.Connectivity) == 0 {
 		return ValidationPending
```

Comment:
@gamli75 should this be Failure as well?

Reply:
yes